### DISCUSSION TITLE
[委托]战略重氮化合物武器测试

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#3024] [其他]
![image](https://staticdelivery.nexusmods.com/mods/2777/images/1770/1770-1705367029-1956102337.jpeg)
### 详细信息:
# 近期 创立者工业(以下简称我司)成功开发出了一种战略级重氮化合物炸弹,但是受限于各种现实因素我司无法得知该型号武器的威力和具体数据,现我司招募三名盗洞客协助我司进行武器测试
## 任务描述:
# 首先 你们将驾驶装备有试验性武器的ADA-01"ADLER"型对地攻击机与三架护航战斗机ADF-01"FALKEN"型飞入零号空洞 然后使用其数据收集功能找到以骸"尼尼微"的所在位置
## 记录完成后离开空洞
# 预计委托报酬:500000丁尼 1600菲林

# *接受委托前请确认您有驾驶类似军用设备的经验
*若损坏公司财产需照价赔偿
*我司不对任务过程中因操作不当造成的风险负责

**Comment by lllliufeng** [comment]
这个委托我接不了╮(╯▽╰)╭

**Comment by ciyuanxingkong** [comment]
这个委托我接不下了

**Comment by Azazel-001** [comment]
可以去找法厄同大佬，大佬无所不能一定可以满你的需求

**Comment by Spr-Aachen** [comment]
v我50，我帮你把吹哥和米老头叫过来
